### PR TITLE
Issue295 a fingerprint and bundle 2

### PR DESCRIPTION
**Pull request by ekr**

### CODE REVIEW

**Comment by ekr** [comment]
@juberti @fluffy Is this approximately right?

**Comment by fluffy** [comment]
Looks close enough to me - lets land this then tune with future PRs

File: draft-ietf-rtcweb-jsep.xml [hunk 1]
```diff
@@ -2367,7 +2348,41 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                         include all of the RID identifiers used in the
                         "a=rid" lines for this m= section.</t>
                     </list></t>
+                    <t>
+                      The following attributes, which are of category
+                      IDENTICAL or TRANSPORT, MUST appear only in "m=" sections which either
+                      have a unique address or which are associated with the bundle-tag.
+                      (In initial offers, this means those "m=" sections which do not
+                      contain an "a=bundle-only" attribute.
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: add closing paren

File: draft-ietf-rtcweb-jsep.xml [hunk 2]
```diff
@@ -3010,6 +2992,45 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                         </list></t>
                     </list></t>
 
+
+                    <t>
+                      Each m= section which is not bundled into another
+                      m= section, MUST contain the following attributes
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: remove extraneous comma

**Pull request merged**